Web UI/CognitiveSearch.UI/appsettings.json

```diff
@@ -5,37 +5,20 @@
       "Default": "Warning"
     }
   },
-
-    // Required fields
-    // "SearchServiceName": "kmcogsearch",
-    // "SearchApiKey": "ECE41FF6473C4D825EBA07908215B22E",
-    // "SearchIndexName": "kmindexv1",
-    // "StorageAccountName": "kmogstorage",
-    // "StorageAccountKey": "JyD+tJlwEcBhFeIZtm2UBZXCHReotDjhXANVO4FeQFl/YhqPjLKe2gfHCUhuxJCnidnKTWjZkGhvBcToAi7o/g==",
-    // "StorageContainerAddress": "https://kmogstorage.blob.core.windows.net/kmdocument",
-    
-    "SearchServiceName": "integtest",
-    "SearchApiKey": "4AA46C618FA0EE606D24E159C657C96B",
-    "SearchIndexName": "indexv3",
-    "StorageAccountName": "prdevurkeastus",
-    "StorageAccountKey": "E/SJVw506Njq/iIN86TJ0BhYxuZi2J5hiTBK18RL6fOnhG/uBlCJWtVITE9WTTc6X97J9v784/Qb9Vtnq5WSQQ==",
-    "StorageContainerAddress": "https://prdevurkeastus.blob.core.windows.net/kmdocs",
-
-    "KeyField": "metadata_storage_path",
-    "IsPathBase64Encoded": true,
-  
-    // Optional instrumentation key
-    "InstrumentationKey": "",
-
-  // Optional container addresses if using more than one indexer:
+  "SearchServiceName": "",
+  "SearchApiKey": "",
+  "SearchIndexName": "",
+  "StorageAccountName": "",
+  "StorageAccountKey": "",
+  "StorageContainerAddress": "",
+  "KeyField": "metadata_storage_path",
+  "IsPathBase64Encoded": true,
+  "InstrumentationKey": "",
   "StorageContainerAddress2": "https://{storage-account-name}.blob.core.windows.net/{container-name}",
   "StorageContainerAddress3": "https://{storage-account-name}.blob.core.windows.net/{container-name}",
-
-  // Optional key to an Azure Maps account if you would like to display the geoLocation field in a map
   "AzureMapsSubscriptionKey": "cz3FUHrarBCljsYSDzp57wRlIG93k4IddPAHIuRyZ7A",
-
-  // Set to the name of a facetable field you would like to represent as a graph.
-  // You may also set to a comma separated list of the facet names if you would like more than one facet type on the graph.
-  "GraphFacet": "keyPhrases, locations"
- 
+  "GraphFacet": "keyPhrases, locations",
+  "ApplicationInsights": {
+    "InstrumentationKey": "0b6b8876-422b-4b3c-9fff-314048b2880d"
```

Comment:
Same with this key, please remove it and replace with a dummy to show as placeholder.

Reply:
Done, replaced with an empty string now.